arch/arm/src/armv7-a/arm_schedulesigaction.c

```diff
@@ -326,13 +329,14 @@ void up_schedule_sigaction(struct tcb_s *tcb, sig_deliver_t sigdeliver)
                    * been delivered.
                    */
 
-                  CURRENT_REGS            =
-                    (FAR void *)STACK_ALIGN_DOWN((uint32_t)CURRENT_REGS -
-                                                 (uint32_t)XCPTCONTEXT_SIZE);
+                  CURRENT_REGS            = (FAR void *)
```

Comment:
Remove all `FAR`?

Reply:
FAR is still used in arm, it's better to remove all FAR in one patch.